### DISCUSSION TITLE
Any way to remove the indents from tab trees?

### DISCUSSION THREAD

**Discussion by ItwasamiiDio** [piroor/treestyletab#3242] [Q&A] [answered]
I've been using the **Auto-show/hide sidebar by mouseover (hover)** code snipet from this section of the Wiki https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules. It's great, but I'd like to remove the indent whenever I make a tree.
![Screenshot 2022-10-28 163428](https://user-images.githubusercontent.com/17091678/198736882-5bb88ab2-38c6-469a-8d94-2cf1869b3f9e.png)
Is there something I can do? I've been experimenting with more code snipets from the Wiki but none seem to do the job.

**Comment by irvinm** [comment, marked as answer]
Just change the base indent from 12 to 0.

![image](https://user-images.githubusercontent.com/979729/198839739-7dddffad-fa79-4111-b52f-aabcb299dc9e.png)
